gradle/lib/dependabot/gradle/file_parser/property_value_finder.rb

```diff
@@ -6,16 +6,19 @@ module Dependabot
   module Gradle
     class FileParser
       class PropertyValueFinder
+        # rubocop:disable Layout/LineLength
+        PROPERTY_DECLARATION_AS_DEFAULTS_REGEX =
+          /(?:\s*(?:project\.findProperty\(\s*['"][^\s]+['"]\s*\)\s*\?|project\.hasProperty\(\s*['"][^\s]+['"]\s*\)\s*\?\s*project\.getProperty\(\s*['"][^\s]+['"]\s*\)\s*):)?/.freeze
```

Comment:
Is there any way we can break this up into smaller chunks? Maybe we could split out the `findProperty`, `hasProperty` and `getProperty` parts into separate variables and concatenate those?

The sheer length of this regex makes it very hard to understand what's going on, for me at least.